app/jobs/socure_docv_results_job.rb

```diff
@@ -6,19 +6,31 @@ class SocureDocvResultsJob < ApplicationJob
   attr_reader :document_capture_session_uuid
 
   # @param [String] document_capture_session_uuid
-  def perform(document_capture_session_uuid:)
+  def perform(document_capture_session_uuid:, async: false)
     @document_capture_session_uuid = document_capture_session_uuid
+    @async = async
 
-    dcs = DocumentCaptureSession.find_by(uuid: document_capture_session_uuid)
-    raise "DocumentCaptureSession not found: #{document_capture_session_uuid}" if !dcs
+    @dcs = DocumentCaptureSession.find_by(uuid: document_capture_session_uuid)
+    raise "DocumentCaptureSession not found: #{document_capture_session_uuid}" if !@dcs
 
-    # analytics = create_analytics(
-    #   user: dcs.user,
-    #   service_provider_issuer: dcs.issuer,
-    # )
+    @analytics = create_analytics(
+      user: @dcs.user,
+      service_provider_issuer: @dcs.issuer,
+    )
+
+    timer = JobHelpers::Timer.new
+    response = timer.time('vendor_request') do
+      socure_document_verification_result
+    end
+    log_verification_request(
+      docv_result_response: response,
+      vendor_request_time_in_ms: timer.results['vendor_request'],
+    )
+    @dcs.store_result_from_response(response)
+  end
 
-    result = socure_document_verification_result
-    dcs.store_result_from_response(result)
+  def perform_later(document_capture_session_uuid:, async: true)
+    perform(document_capture_session_uuid: document_capture_session_uuid, async: async)
```

Comment:
```suggestion
```

i think we just need to pass argument async: true [here](https://github.com/18F/identity-idp/blob/78da74951b7c7c2ccb3afdcf0f1b366db665021d/app/controllers/socure_webhook_controller.rb#L41)

Reply:
@amirbey shouldn't we be defaulting to true and then during the perform_now we would set async=false ?